.circleci/config.yml

```diff
@@ -126,15 +125,18 @@ jobs:
             VERSION=$(./bin/driftctl_linux_amd64 version)
             ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -n ${VERSION} ${VERSION} ./bin/
   update-lambda:
-    executor: aws-cli/default
     environment:
         FUNCTION_NAME: driftctl-version
+    docker:
+        - image: cimg/base:2021.04
     steps:
-      - aws-cli/install
       - run:
           name: "Update Lambda version"
           command: |
-            aws lambda update-function-configuration --function-name $FUNCTION_NAME --environment "{\"Variables\":{\"LATEST_VERSION\":\"$CIRCLE_TAG\"}}"
+              wget "https://github.com/cloudskiff/lambda-env-updater/releases/download/v1.0.0/lambda-env-updater_linux_amd64" && chmod +x lambda-env-updater_linux_amd64
+              ./lambda-env-updater_linux_amd64\
```

Comment:
`./lambda-env-updater_linux_amd64 \`

Reply:
Why adding an additional space there ?

Reply:
Because otherwise it makes the shell asking for an input (even with bash)
```
 $ ./bin/driftctl_linux_amd64\
> 
```

Reply:
Well in your diff there's a space in the next line so it'll work anyway, nevermind 😄